app/src/test/kotlin/io/homeassistant/companion/android/webview/addto/EntityAddToHandlerTest.kt

```diff
@@ -237,6 +239,70 @@ class EntityAddToHandlerTest {
         verify { TodoWidgetConfigureActivity.newInstance(context, entityId) }
     }
 
+    @ParameterizedTest
+    @ValueSource(
+        strings = [
+            "light.test",
+            "alarm_control_panel.test",
+        ],
+    )
+    fun `Given standard entity on automotive when getting actionsForEntity then returns auto favorite`(entityId: String) = runTest {
+        mockGetEntity(entityId)
+
+        val actions = handler.actionsForEntity(isAutomotive = true, isQuest = false, entityId)
+
+        assertEquals(listOf(EntityAddToAction.AndroidAutoFavorite), actions)
+    }
+
+    @ParameterizedTest
+    @ValueSource(
+        strings = [
+            "$MEDIA_PLAYER_DOMAIN.test",
+            "$TODO_DOMAIN.test",
+            "$CAMERA_DOMAIN.test",
+            "$IMAGE_DOMAIN.test",
+        ],
+    )
+    fun `Given entity on automotive not vehicle domain with when getting actionsForEntity then returns empty list`(entityId: String) = runTest {
+        mockGetEntity(entityId)
+
+        val actions = handler.actionsForEntity(isAutomotive = true, isQuest = false, entityId)
+
+        assertEquals(emptyList<EntityAddToAction>(), actions)
+    }
+
+    @ParameterizedTest
+    @ValueSource(
+        strings = [
+            "light.test",
+            "alarm_control_panel.test",
+        ],
+    )
+    fun `Given standard entity on Quest when getting actionsForEntity then returns auto favorite`(entityId: String) = runTest {
+        mockGetEntity(entityId)
+
+        val actions = handler.actionsForEntity(isAutomotive = false, isQuest = true, entityId)
+
+        assertEquals(listOf(EntityAddToAction.AndroidAutoFavorite), actions)
```

Comment:
This test may technically work but you realistically won't have Android Auto on a Meta Quest (supposed to use minimal flavor, no Google services)

Reply:
> Indeed, in that case are you ok merging like it is today?

Yes

> For the AndroidAuto we could check if we are on a minimal build when not automotive?

The check should be inverted, [like](https://github.com/home-assistant/android/blob/a6ed21cdb08b7a20f4f6cbe8f242031388d079f8/app/src/main/kotlin/io/homeassistant/companion/android/settings/SettingsFragment.kt#L385-L387) in settings: flavor is full (or system is Automotive) (and also check SDK level)